src/test/resources/application-dev.yml

```diff
@@ -53,7 +53,7 @@ dexi:
     username: dexi_username
     password: dexi_password
 amedia:
-  url: http:/localhost/api/pipek/v2/classified/search
+  url: http:/localhost/api/prophecy/external/nav/jobs
```

Comment:
Få med `amedia.apiKey` her og fortelle utvikler om at det må fylles inn noe fornuftig for å kunne jobbe med API-et lokalt ? `<API token here>`

Reply:
Tenker ca:
```
amedia:
  [...]
  apiKey: ${AMEDIA_API_KEY:need-to-set-token}
  [...]
```
Da kan utvikler bare sette nøkkelen i en miljø-variabel, f.eks.